### DISCUSSION TITLE
Is missing some FasmG setting?

### DISCUSSION THREAD

**Discussion by ASMHSE** [bitRAKE/UEFI_playground#1] [Q&A] [answered]
Trying to buid examples (build_to_image.cmd) I obtain:
 
fyi: relocation data was not needed
5_mem_map.asm [1]:
        include 'UEFI64_app.inc'
 N:\UEFI_playground\include/UEFI64_app.inc [10]:
        format pe64 efi on 'nul'
macro PE64 [44]:
        format binary as '.efi'
Processed: format binary as '.efi'
Error: invalid argument.

Thanks in advance.

**Comment by bitRAKE** [comment, marked as answer]
Sorry, I forgot that I use [a modified version of fasmg](https://board.flatassembler.net/topic.php?p=218441#218441). If you just remove the "." period it should work with baseline fasmg. Note this modified version of fasmg would be a security risk - I'm just okay with that.

**Reply by ASMHSE** [reply]
Perfect with fasmg-experimental. Thanks.
